tensorflow/python/data/experimental/ops/error_ops.py

```diff
@@ -20,10 +20,10 @@
 from tensorflow.python.data.ops import dataset_ops
 from tensorflow.python.ops import gen_experimental_dataset_ops
 from tensorflow.python.util.tf_export import tf_export
-
+from tensorflow.python.compat import compat
 
 @tf_export("data.experimental.ignore_errors")
-def ignore_errors():
+def ignore_errors(log_warning=False):
```

Comment:
You will need to regenerate the golden API files as your change is changing the public API.

You can do so by running:

```
$ bazel build tensorflow/tools/api/tests:api_compatibility_test
$ bazel-bin/tensorflow/tools/api/tests/api_compatibility_test --update_goldens True
```

Reply:
In addition, the docstring should be updated with an "Args:" section (see other source files for examples) that describe the purpose of the argument.

Reply:
Thanks for your comment. The docstring is updated with an "Args:" section. The golden API files is also updated.